gradle.properties

```diff
@@ -3,7 +3,7 @@ org.gradle.jvmargs=-Xmx2G
 # Fabric Properties
 # check these on https://fabricmc.net/use
 minecraft_version=1.16.5
-yarn_mappings=1.16.5+build.5
+yarn_mappings=1.16.5+build.6
```

Comment:
Just wondering why this is necessary, but everything else looks great!

Reply:
Ah, that’s just a leftover from my Linux setup. For some reason build 5 gives an unexplainable error when trying to use runClient on my Arch setup, and build 6 doesn’t have any conflicts anyway so I don’t see any reason not to.